### ISSUE TITLE
The blog can not be viewed in english

### ISSUE THREAD

**Issue by Dbz** [lale-help/lale-help#404]
When on the [blog](http://lale.help/blog), if you attempt to change the language from German to English, it takes you back to the home page. When you go back to the blog it is again in German.

**Comment by Dbz** [comment]
This looks like it could be a bug with the English/German toggle in general where it takes you back to the home page

**Comment by pjfanning** [comment]
I tried http://lale.help/home_en and the News link in the top banner goes to http://lale.help/blog which is in German and a heading of Tutorials.

**Comment by dottorer** [comment]
The slug on the 'tutorials' page is '/blog' since that what is was originally. I did not want to break external links so we kept it. 

The problem with EN/DE for the blog is that we have not found a good way to make it properly i18n. Most posts for now are targeted to DE audience so we kept it as German only. We decided to have it appear in the EN nav for completeness. We might also post EN content as we go forward.

The issue with the language toggle is know, but I do not know how to build it in webflow in a way that it is part of the re-usable header element and also jumps to the EN version of the respective page. Any ideas on how to do this?
